packages/phasor/src/utils/util.ts

```diff
@@ -0,0 +1,12 @@
+export const EPSILON = 1e-12;
```

Comment:
Seems that this module is not being used? And we have an `almostEqual`, though it's located in the `blocks` package and should not be imported by phasor, though.

Reply:
yeah, actually I was intended to add Point class with more methods which will use functions in this module. 
but I figure that one PR better focus on one thing. So I didn't include more methods.
So, I think this time I can drop this module.